hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java

```diff
@@ -315,6 +318,27 @@ public boolean isRunAllowed(Operation op) {
     }
 
     @Override
+    public <E> Set<InternalCompletableFuture<E>> invokeOnCluster(OperationFactory operationFactory,
+                                                                 MemberSelector memberSelector) {
+        Collection<Member> members;
+        if (memberSelector == null) {
+            members = nodeEngine.getClusterService().getMembers();
+        } else {
+            members = nodeEngine.getClusterService().getMembers(memberSelector);
+        }
+
+        Set<InternalCompletableFuture<E>> futures = new HashSet<InternalCompletableFuture<E>>(members.size());
+
+        for (Member member : members) {
+            Address target = member.getAddress();
+            Operation operation = operationFactory.createOperation();
+            InternalCompletableFuture<E> future = invokeOnTarget(operation.getServiceName(), operation, target);
```

Comment:
Does this impl conforms the contract in javadoc?
> In case of failure to invoke the operation on a member, no attempt is made to continue invoking the operations on other members

Currently, operation is submitted to all members.